assignments/lambda-classes.js

```diff
@@ -1 +1,194 @@
 // CODE here for your Lambda Classes
+//Person Object Creator
+class Person{
+    constructor(attributes){
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+    speak(){
+       return `Hello my name is ${this.name}, I am from ${this.location}.`;
+    }
+}
+// Children of Person Creator
+
+// Instructor Creator
+class Instructor extends Person{
+    constructor(instructorAttributes){
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+    demo(subject){
+        return `Today we are learning about ${subject}`;
+    }
+    grade(student,subject){
+        return `${student.name} receives a perfect score on ${subject}`;
+    }
+    grader(student){
+       const posOrNeg = RNG(1,100);
+       
+            if (posOrNeg >= 50){
+                student.grade = student.grade + RNG(1,10); 
+            }
+           else if (posOrNeg < 50){
+                student.grade = student.grade - RNG(1,10);
+            }
+          
+    }
+}
+
+// Student Creator
+class Student extends Person{
+    constructor(studentAttributes){
+        super(studentAttributes);
+        const startingGrade = RNG(75,100);
+        this.previousBackground = studentAttributes.previousBackground;
+        this.className = studentAttributes.className;
+        this.favSubjects = studentAttributes.favSubjects;
+        this.grade = startingGrade
+    }
+    listsSubjects(){
+          this.favSubjects.forEach(function (subject){
+              console.log(subject)
+          }); 
+            
+        }
```

Comment:
The spacing and indentation are a little off here; prettier could probably do the work of fixing that for you.